### PR TITLE
Integration with Github pages [Docusaurus]

### PR DESCRIPTION
**Pull request by ssnickolay**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+- 'rubocop-md'
 AllCops:
   Exclude:
     - 'bin/**/*'
     - 'tmp/**/*'
+    - 'docs/web/**/*'
     - 'vendor/**/*'
     - 'gemfiles/vendor/**/*'
+    - 'CHANGELOG.md'
+    - 'README.md'
```
Comment (palkan) [inline .rubocop.yml]: Why not?

File: gemfiles/railsmaster.gemfile [hunk 2]
```diff
@@ -3,5 +3,6 @@ source 'https://rubygems.org'
 gem 'arel', github: 'rails/arel'
 gem 'rails', github: 'rails/rails'
 gem 'sqlite3'
+gem 'rubocop-md'
```
Comment (palkan) [inline gemfiles/railsmaster.gemfile]: Let's add it to `gemspec` instead of adding to every gemfile

File: .rubocop.yml (hunk 1 above)
Comment (ssnickolay) [reply .rubocop.yml]: I thought that we would get rid of ruby code in README and now it does not make sense to fix this.

File: gemfiles/railsmaster.gemfile (hunk 2 above)
Comment (ssnickolay) [reply gemfiles/railsmaster.gemfile]: It sounds good
https://github.com/palkan/clowne/pull/10/commits/50e783099369bbe86cbe75cf605744e6fe62bdd4

File: .rubocop.yml (hunk 1 above)
Comment (ssnickolay) [reply .rubocop.yml]: But some code in the README will remain, so
https://github.com/palkan/clowne/pull/10/commits/c0b57cfae48ad6a63ebf6ab25a038c90ccdc9d80

**Pull request merged**